### PR TITLE
Improve design desktop/mobile

### PR DESCRIPTION
**Pull request by zyhou**
Peek 2019-12-18 14-49

### CODE REVIEW

**Comment by Euregan** [comment]
#### 📝Changed routes:


| [**/**](https://aperoweb-git-design.euregan.now.sh/) | [**/talks**](https://aperoweb-git-design.euregan.now.sh/talks) |
|:-:|:-:|
|<a href="https://aperoweb-git-design.euregan.now.sh/">Screenshot of /</a><br /><sup><a href="https://images.weserv.nl?url=microlink.nyc3.digitaloceanspaces.com/fy1W12e2T_uyjtyyxf9HqSdZ_WI_">(view full size)</a> |<a href="https://aperoweb-git-design.euregan.now.sh/talks">Screenshot of /talks</a><br /><sup><a href="https://images.weserv.nl?url=microlink.nyc3.digitaloceanspaces.com/_yyJp1W5NA_z2kL_KU3dDF1O92A_">(view full size)</a> |







Commit 28dc59b58c250273c33c8c1111c061b859ee6fa5 (https://aperoweb-git-design.euregan.now.sh).

File: assets/styles.less [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    list-style: none;
+    padding: 0;
+}
+
+.ant-layout-content {
+    padding: 0 2rem;
+}
+
+.ant-card {
+    width: 300px;
```
Comment (Euregan) [inline assets/styles.less]: Don't use pixels, use `rem`

Comment (Euregan) [reply assets/styles.less]: Also, min/max rather than a fixed width maybe?

File: components/Layout.js [hunk 2]
```diff
@@ -12,34 +13,6 @@ const CustomLayout = ({ children }) => (
...
-          height: auto !important;
-          padding: 0 !important;
-        }
-
-        .ant-card .ant-card-body > * ~ * {
-          margin-top: 0.5rem;
-        }
-
-        h2 {
-          margin-top: 1rem !important;
```
Comment (Euregan) [inline components/Layout.js:31]: The `margin-top` is missing

**Review by Euregan — CHANGES_REQUESTED** [review]
You also need to delete the component `Grid.js` since it's not used anymore

File: components/Layout.js (hunk 2 above)
Comment (zyhou) [reply components/Layout.js:31]: No need anymore. The h3 title already have `margin-bottom: 0.5em;`

**Comment by zyhou** [comment]
> You also need to delete the component `Grid.js` since it's not used anymore

The file is delete

File: assets/styles.less (hunk 1 above)
Comment (zyhou) [reply assets/styles.less]: I prefer when all card have same size

File: components/Layout.js (hunk 2 above)
Comment (Euregan) [reply components/Layout.js:31]: There is a need, the "Communication" title is too close to the "Talks" cards

File: assets/styles.less (hunk 1 above)
Comment (Euregan) [reply assets/styles.less]: Ok then, but a grid with different breakpoints would have been better than using flex then

File: components/Layout.js (hunk 2 above)
Comment (zyhou) [reply components/Layout.js:31]: image

**Review by Euregan — APPROVED** [review]

**Pull request merged**